launch-dedicated.sh

```diff
@@ -23,7 +28,12 @@ ADVERTISE_ONLINE="${AdvertiseOnline:-"True"}"
 ENABLE_SINGLE_PLAYER="${EnableSingleplayer:-"False"}"
 PASSWORD="${Password:-""}"
 
-cd engine
+cd "${TEMPLATE_ROOT}"
+if [ ! -d "${ENGINE_DIRECTORY}" ]; then
+	make -f Makefile
```

Comment:
Ah, now I get you.  You are referring to the `-f Makefile`, right?  that was a leftover from an older version where I was `-f ${TEMPLATE_ROOT}/Makefile`ing.